controllers/hosting/hostingdeployment_controller_test.go

```diff
@@ -217,15 +217,11 @@ var _ = Describe("Reconciling a HostingDeployment that exists", func() {
 	})
 
 	Context("Endpoint does not exist", func() {
```

Comment:
Here we mean does not exist in k8s? update message to be specific

Reply:
Agree, this wording was confusing for me too. I'll fix these